src/get-dart/index.md

```diff
@@ -1,39 +1,66 @@
 ---
 title: Get the Dart SDK
 description: Get the libraries and command-line tools that you need to develop Dart web, command-line, and server apps.
+channel-list: [Stable, Beta, Dev]
 js:
 - url: /assets/js/get-dart/install.js
   defer: true
 ---
 
 This page describes how to download the Dart SDK.
-The Dart SDK has the libraries and command-line tools that you need to develop
-Dart command-line, server, and non-Flutter web apps.
-For details, see the [Dart SDK overview](/tools/sdk).
-
-## Installing the Dart SDK {#install}
-
-As the following instructions show,
-you can use a package manager
-to easily install and update a stable channel Dart SDK.
-Alternatively, you can
-[build the SDK from source][],
-grab a [Dart Docker image][], or
-install from [any release channel](#release-channels) by
-[downloading the SDK as a zip file][].
+The Dart SDK includes the libraries and command-line tools that you need to develop Dart command-line, server, and non-Flutter web apps.
+
+To learn more about the Dart SDK, see the [Dart SDK overview](/tools/sdk).
+
+## System requirements
+
+Dart supports the following hardware architectures and platform versions
+to develop and run Dart code.
+
+{% assign yes = '<span class="material-icons" style="color: #158477;">verified</span>' %}
+{% assign no = '<span class="material-icons" style="color: #D43324">dangerous</span>' %}
+{% assign beta = '<span class="material-icons" style="color: #13C2AD">gpp_maybe</span>' %}
+{% assign macversions = '' %}
+{% for version in site.data.macos limit:3 %}
+{%- if version.eol == false -%}
+{% assign macversions = macversions | append: version.cycle | append: ' (' | append: version.codename  | append: ')' %}
+{%- unless forloop.last -%}{% assign macversions = macversions | append: ', ' %}{% endunless -%}
+{%- endif %}
+{% endfor %}
+
+| Platform |   x86   |   x64   |   ARM   |   ARM64   | RISC-V    | OS Versions                              |
+|----------|---------|---------|---------|-----------|-----------|------------------------------------------|
+| Windows  | {{yes}} | {{yes}} | {{no}}  | {{beta}} | {{no}}    | 10 (32-bit, 64-bit), 11                                   |
+| Linux    | {{yes}} | {{yes}} | {{yes}} | {{yes}}   | {{beta}} | [Debian stable][], [Ubuntu LTS][]        |
+| macOS    | {{no}}  | {{yes}} | {{no}}  | {{yes}}   | {{no}}    | {{macversions}} |
+{:.table .table-striped}
+
+{{beta}} Supported in Dev, Beta channels only.
+
+## Choose an install option
+
+To install and update the Dart SDK from the stable channel,
+choose one of the following options:
+
+1. [Use a package manager](#install) (Recommended).
+1. [Build the SDK from source][build-source]
```

Comment:
Building from source should be last on the list (it's not really intended for users of Dart, only for those few that contribute to the SDK)